examples/angular-cli/package.json

```diff
@@ -13,20 +13,22 @@
     "test": "ng test"
   },
   "dependencies": {
-    "@angular/common": "^5.2.10",
-    "@angular/compiler": "^5.2.10",
-    "@angular/core": "^5.2.10",
-    "@angular/forms": "^5.2.10",
-    "@angular/platform-browser": "^5.2.10",
-    "@angular/platform-browser-dynamic": "^5.2.10",
-    "@ngrx/store": "^5.2.0",
+    "@angular/common": "^6.0.0-rc.5",
```

Comment:
As mentioned above, new angular/cli v6 needs not `.angular-cli.json` but `angular.json`. I think it's better to re-generate the example project with `ng new angular-cli`.

Reply:
Stable 6 version is out now, please upgrade